.gitignore

```diff
@@ -3,3 +3,4 @@ ENV
 dist
 1pass.egg-info
 build
+.idea
```

Comment:
Where'd .idea come from?

Reply:
Sorry, that's for PyCharm, I guess I should add it to my global gitignore